src/ansys/speos/core/generic/general_methods.py

```diff
@@ -108,3 +114,68 @@ def wrapper(*args, **kwargs):
         return method(*args, **kwargs)
 
     return wrapper
+
+
+def error_no_install(install_path: Union[Path, str], version: Union[int, str]):
+    """Raise error that installation was not found at a location.
+
+    Parameters
+    ----------
+    install_path : Union[Path, str]
+        Installation Path
+    version : Union[int, str]
+        Version
+    """
+    if not install_path:
+        install_loc_msg = ""
+    else:
+        install_loc_msg = f"at {Path(install_path).parent}"
```

Comment:
```suggestion
    install_loc_msg = ""
    if install_path:
        install_loc_msg = f"at {Path(install_path).parent}"
```
I know I was the one to propose this changes but I think this would enhance it even more.